tools/parity_scale_decoder/Cargo.toml

```diff
@@ -13,20 +13,22 @@ license.workspace = true
 no-color = ["colored/no-color"]
 
 [dependencies]
-iroha_data_model = { version = "=2.0.0-pre-rc.16", path = "../../data_model", features = ["http"]}
-iroha_primitives = { version = "=2.0.0-pre-rc.16", path = "../../primitives", default-features = false }
-iroha_schema = { version = "=2.0.0-pre-rc.16", path = "../../schema"}
-iroha_schema_gen = { version = "=2.0.0-pre-rc.16", path = "../../schema/gen"}
-iroha_crypto = { version = "=2.0.0-pre-rc.16", path = "../../crypto", default-features = false }
-iroha_version = { version = "=2.0.0-pre-rc.16", path = "../../version", default-features = false }
-iroha_genesis = { version = "=2.0.0-pre-rc.16", path = "../../genesis"}
-clap = { version = "3.2.23", features = ["derive", "cargo"] }
-eyre = "0.6.8"
-parity-scale-codec = { version = "3.2.1", default-features = false }
+iroha_data_model = { workspace = true, features = ["http"] }
+iroha_primitives = { workspace = true }
+iroha_schema = { workspace = true }
+iroha_schema_gen = { workspace = true }
+iroha_crypto = { workspace = true }
+iroha_version = { workspace = true }
+iroha_genesis = { workspace = true }
+
+clap = { workspace = true, features = ["derive", "cargo"] }
+eyre = { workspace = true }
+parity-scale-codec = { workspace = true }
 colored = "2.0.0"
 
 [build-dependencies]
-iroha_data_model = { version = "=2.0.0-pre-rc.16", path = "../../data_model"}
-parity-scale-codec = { version = "3.2.1", default-features = false }
+iroha_data_model = { workspace = true }
+
+parity-scale-codec = { workspace = true }
 serde_json = "1.0.91"
 serde = "1.0.151"
```

Comment:
seems to be missing here

Reply:
I _think_ it's fine?

I just added an empty line between `iroha_*` deps and other deps, like is done in most other `Cargo.toml`s